frontend/src/components/SiteNav.js

```diff
@@ -89,7 +89,7 @@ const SiteNav = ({
                       Activity Reports
                     </NavLink>
                   </li>
-                  <FeatureFlag flag="training_reports">
+                  <FeatureFlag flag="training_reports" renderNotFound>
```

Comment:
Do we want a not found render in the links?

![image](https://github.com/HHS/Head-Start-TTADP/assets/84350609/644154e5-0342-4918-bfee-cbb315ed82d3)